connect/v1.0/connect.md

```diff
@@ -57,8 +57,12 @@ Finally, we convert the binary into a base64 string to create the final signatur
 
 ClickView will perform a HTTP `post` to the provided `redirect_uri`. The body will contain a `application/x-www-form-urlencoded` response. The data included in this response is detailed [below](#data).
 
+The response will also contain a signature of the response parameters (excluding the signature). This will allow you to verify the payload being posted back to you. To verify the signature please see above.
```

Comment:
The example above has the response parameters are lower snake case, is it the same for the response (so `school_id` instead of `schoolId` or `SchoolId` or `schoolid`)?